importer/templates/importer/upload.html

```diff
@@ -0,0 +1,6 @@
+<h1>upload here</h1>
```

Comment:
You can just use the one I put in the cookiecutter that you reviewed: https://github.com/mitodl/cookiecutter-djangoapp/blob/master/%7B%7B%20cookiecutter.app_name%20%7D%7D/%7B%7B%20cookiecutter.app_name%20%7D%7D/templates/base.html